CHANGES.md

```diff
@@ -1,5 +1,15 @@
 # Change Log
 
+### 1.75 - 2020-09-08
+
+##### Additions :tada:
```

Comment:
I think that it would make sense to leave only the **Additions** entry as your contribution contains changes that *add* something to the public API. What I mean is that it was not something broken and your code fixed it 😉

Reply:
Ah, great point.  I'll clean that up.